api/profile/src/config.rs

```diff
@@ -0,0 +1,16 @@
+use spin_sdk::config::get;
+
+const KEY_DB_URL: &str = "db_url";
+
+#[derive(Debug)]
+pub(crate) struct Config {
+    pub db_url: String,
+}
+
+impl Config {
+    pub(crate) fn get() -> Config {
```

Comment:
Maybe `impl Default for Config`. `Config::get` is a little confusing when paired with `spin_sdk::config::get`.

Reply:
Is that essentially the rust equivalent of a parameterless constructor? Coming from a Java/.NET world. Really just wanted one place where I can call `spin_sdk::config::get`.

Reply:
Exactly. I think having `Config::default` is just slightly more idiomatic.